src/test/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersResponseTest.java

```diff
@@ -20,10 +17,5 @@ public void professionalUsersTest() throws NoSuchFieldException, IllegalAccessEx
 
         String email = "";
 
-        Field f = professionalUsersResponse.getClass().getDeclaredField("email");
-        f.setAccessible(true);
-        email = (String) f.get(professionalUsersResponse);
-
-        assertEquals(email, expectEmailAddress);
```

Comment:
Please remove the above class

Reply:
Ok, good spot. I have fixed the test instead by adding Lombock @Getter. As long as all the fields are final I do not have a problem with doing it this way due to the fact that using reflection will break the Travis build.

Reply:
See above. We need NOT be removing test classes. This will be left here and can be improved when there is time to do so.